src/test/Components/Blueprints/Blueprints.test.tsx

```diff
@@ -104,6 +107,43 @@ describe('Blueprints', () => {
     expect(buildImageBtn).toBeEnabled();
   });
 
+  test('uncheck Target option and check that build image button is Disable', async () => {
+    renderWithReduxRouter('', {});
+
+    await selectBlueprintById(blueprintIdWithComposes);
+    const buildImageBtn = await screen.findByRole('button', {
+      name: /Build image/i,
+    });
+    expect(buildImageBtn).toBeEnabled();
+    const buildImageDropDown = screen.getByTestId('blueprint-build-image-menu');
+    await user.click(buildImageDropDown);
+    const awsOption = screen.getByText(/amazon web services \(aws\)/i);
+    await user.click(awsOption);
+    const buildSelectedBtn = screen.getByRole('button', {
+      name: /build selected/i,
+    });
+    expect(buildSelectedBtn).toBeDisabled();
+  });
+
+  test('uncheck one Target option and check that you can build an image', async () => {
```

Comment:
The test doesn't seem to test what it says it's testing, or does it?
I see expect on button to be disabled, that's probably not correct, right?